packages/lesmis/src/views/DashboardView.js

```diff
@@ -50,18 +50,41 @@ const makeTabOptions = entityType => [
     label: entityType ? `${entityType} Profile` : 'Profile',
     Component: DashboardReportTabView,
   },
+  {
+    value: 'essdpPlan',
+    label: 'ESSDP Plan 2021-25 M&E Framework',
+    Component: TabTemplate,
+    Body: '9th Education Sector and Sports Development Plan 2021-25 M&E Framework',
+  },
   {
     value: 'earlyChildhood',
-    label: 'ESSDP Early Childhood Education',
+    label: 'ESSDP Early childhood education sub-sector',
+    Component: TabTemplate,
+    Body: 'ESSDP Early childhood education sub-sector',
+  },
+  {
+    value: 'primary',
+    label: 'ESSDP Primary sub-sector',
+    Component: TabTemplate,
+    Body: 'ESSDP Primary sub-sector',
+  },
+  {
+    value: 'secondary',
+    label: 'ESSDP Lower secondary sub-sector',
+    Component: TabTemplate,
+    Body: 'ESSDP Lower secondary sub-sector',
+  },
+  {
+    value: 'emergency',
+    label: 'Emergency in Education/COVID-19',
     Component: TabTemplate,
-    Body: 'ESSDP Early Childhood Education',
+    Body: 'Emergency in Education/COVID-19',
   },
-  { value: 'primary', label: 'ESSDP Primary', Component: TabTemplate, Body: 'ESSDP Primary' },
   {
-    value: 'indicatorSelection',
-    label: 'Free Indicator Selection',
+    value: 'international',
+    label: 'International reporting on SDGs',
     Component: TabTemplate,
-    Body: 'Free Indicator Selection',
+    Body: 'International reporting on SDGs',
```

Comment:
Oh I think we need the indicator selection on top of the ones Erin listed thanks